packages/interbit/package.json

```diff
@@ -13,6 +13,7 @@
     "deep-diff": "^1.0.1",
     "fs-extra": "^5.0.0",
     "interbit-core": "0.14.0",
+    "interbit-root-covenant": "0.4.43",
     "interbit-covenant-tools": "0.4.44",
```

Comment:
Mismatched versions. These should be the same. Currently 0.4.46

Reply:
Merged it in :+1: